### DISCUSSION TITLE
Welcome to Auto-GPT-sandbox-wizard Discussions!

### DISCUSSION THREAD

**Discussion by rgalloni** [rgalloni/Auto-GPT-sandbox-wizard#13] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by DarkUser089** [comment]
Hi, so after i install it and it works, how do i start it everytime? Do i start the install.ps1 every time?

Thanks and sry for the stupid question

**Reply by rgalloni** [reply]
Hello @DarkUser089, correct. That's if you want to use the auto-update feature on start-up.

**Reply by DarkUser089** [reply]
Hi @rgalloni thx a lot for the fast answer. Other little question, how can i solve this?:

"Warning: The file 'auto-gpt.json' does not exist. Local memory would not be saved to a file."

i creates the auto-gpt.json in the Auto-GPT Folder but i still get this message.

**Comment by Andrew68IT** [comment]
Windows install.
I get this error:
Can you help?
Thanks

 .\auto_gpt_easy_install.ps1
.\auto_gpt_easy_install.ps1 : Impossibile caricare il file C:\Users\andre\downloads\auto_gpt_easy_install.ps1.
L'esecuzione di script è disabilitata nel sistema in uso. Per ulteriori informazioni, vedere about_Execution_Policies
all'indirizzo https://go.microsoft.com/fwlink/?LinkID=135170.
In riga:1 car:1
+ .\auto_gpt_easy_install.ps1
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : Errore di protezione: (:) [], PSSecurityException
    + FullyQualifiedErrorId : UnauthorizedAccess

**Reply by DarkUser089** [reply]
Open Powershell as Administrator and then do this:

`Set-ExecutionPolicy RemoteSigned`
